experimentation/image_compression.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import pathlib
+import cv2
+import imagej
+import skimage
+import sys
+import os 
+import matplotlib.pyplot as plt
+import numpy as np
+from IPython.utils.io import capture_output
+import warnings
+from PIL import Image
+sys.path.append(os.path.abspath('./'))
+from processor import BasicpyPreprocessor
+import PyBaSiC.pybasic
+
+fiji_path = pathlib.Path("/home/keenanmanpearl/Desktop/mitocheck/fiji-linux64/Fiji.app")
+movie_path = pathlib.Path(
+    "/home/keenanmanpearl/Desktop/mitocheck_movies/movies/LT0001_02/1.ch5/00001_01.ch5"
+)
+fiji = BasicpyPreprocessor(fiji_path)
+movie = fiji.load_mitocheck_movie_data(movie_path)
+corrected_movie = fiji.pybasic_illumination_correction(movie)
+
+
+
+def process_movie(down_factor, height, width, save_path, fourcc, fps, frames, corrected_movie):
+    down_height = int(height / down_factor)
+    down_width = int(width / down_factor)
+    down_points = (down_width, down_height)
+    # False for black and white
+    out = cv2.VideoWriter(f"{save_path}", fourcc, fps, down_points, False)
```

Comment:
instead of position-based function arguments, can you name the arguments?

https://www.pythonmorsels.com/positional-vs-keyword-arguments/

> Named arguments can sometimes make your code a bit more descriptive because you've given a name to an object whose use might not otherwise be clear simply by its position in a function call.

Reply:
please use named arguments everywhere! It is so helpful for readability :)

Reply:
(even for functions you yourself write!)